### ISSUE TITLE
webkit2 network settings

### ISSUE THREAD

**Issue by keis** [uzbl/uzbl#371]
From the list of removed variables in #225 there's quite a few network related options that would be good to bring back.

proxy support made a comeback in 2.16 https://webkitgtk.org/reference/webkit2gtk/unstable/WebKitNetworkProxySettings.html

**Comment by mathstuf** [comment]
Yeah, I have a git repo with the history of the webkit releases that I need to catch up on. There are also deprecation warnings with newer webkit2 APIs as well.
